### DISCUSSION TITLE
Add tags to better describe, filter and discover scripts on the website

### DISCUSSION THREAD

**Discussion by DevJulian1966** [community-scripts/ProxmoxVE#220] [Ideas]
I really like the design of the new website, i.e. the dedicated "Website" button which brings you to the project's site makes it way clearer.
One thing I really miss are tags. When searching for ebook related services, let's have a tag called "ebook" which maybe shows scripts from different categories. For inventory management we'll maybe set "inventory" as well as "management" and so on.
Especially if you don't get the name of a service right, it's very hard to find.

**Comment by MickLesk** [comment]
@community-scripts/frontend-dev Team, what do you say? It makes sence for me. Example: I search for "code" and say vs Code, pastebin, zipline, ...

**Reply by bramsuurdje** [reply]
yup, agree. we can put it on the todo list

**Comment by Nelluk** [comment]
Not sure if this request should be a new thread, but I was hoping to see another search improvement so that the description of a script could match a search. For example if I search "finance" there are no results, even though there is at least one script with "finance" in the description.
